MEN-6086: docs: basic readme content (#9)
mendersoftware/app-update-module: README.md

# Mender Application Updates

> The repo contains Mender Update Modules for supporting containerized application updates on devices.

Reviewer: ```suggestion
The repo contains Mender Update Module and artifact generator
```

Author: @merlin-northern what are the plans for future? Will the repo contain only those or it's a plans for something more?

Reviewer: at one point we will provide support for applications based on something else than containers. for now we focus on container-based deployments.
